index.html

```diff
@@ -36,14 +36,42 @@
 
 <p>This project is an implementation of the paper "Meshing piecewise linear complex using CDT" by Hang Si et. al. Constrained Delaunay tetrahedralization attempts to retain Delunay property for cells while ensuring preservation of constraint vertices, segments and facets. </p>
 
+<h1>
+<a id="terminology" class="anchor" href="#terminology" aria-hidden="true"><span class="octicon octicon-link"></span></a>Terminology</h1>
+
+<p>Delaunay: Let V be the set of vertices of X(a PLC). Let σ be any simplex formed by vertices of V . σ is delaunay if there exists a circumsphere of σ that encloses no vertex of V.</p>
+
+<p>Edge Protected: X is edge-protected if every segment of X is strongly delaunay.</p>
+
+<p>Delaunay tetrahedralization: A Delaunay tetrahedralization D of V is a tetrahedralization such that all simplices of D are delaunay.</p>
+
+<p>Contrained Delaunay: A tetrahedron t formed by vertices of X is constrained delaunay if its circumsphere encloses no vertex of X which is visible from any point in the relative interior of t.</p>
+
+<p>Constrained tetrahedralization: A tetrahedralization T is a constrained tetrahedralization of X if T and X have exactly the same vertices, and the tetrahedra in T cover the convex hull of V . every segment of X is an edge of T , every facet of X is a union of triangular faces of T .</p>
+
+<p>CDT: A constrained tetrahedralization T of X is said to be a constrained delaunay tetrahedralization (CDT) of X if each tetrahedron of T is constrained delaunay.</p>
+
+<p>Local Degenracy: Let D be the delaunay tetrahedralization of the vertices of X, t a tetrahedron in D and t' an adjacent tetrahedron of t (sharing a face with t), V the set of vertices of t, t'. If all vertices of V lie on a common sphere, V is called a local degeneracy of X.</p>
+
+<h1>
+<a id="the-two-alternatives" class="anchor" href="#the-two-alternatives" aria-hidden="true"><span class="octicon octicon-link"></span></a>The Two alternatives</h1>
+
+<p>The paper highlights two ways to get to be sure that a CDT exists.The implementation is done using the second alternative because using second alternative the facet recovery(last step) can be carried out efficiently.</p>
+
```

Comment:
@HarshitJn IMO you can add the line "The implementation is done using the second alternative because using second alternative the facet recovery(last step) can be carried out efficiently."  after both approaches have been introduced so that reader can reason why we have chosen second approach. Further, notice that reader will not be able to understand the term 'facet recovery'...so maybe a bit of explanation may be required while mentioning second approach.